bower.json

```diff
@@ -20,7 +20,10 @@
   ],
   "dependencies": {
     "polymer": "^1.6.0",
-    "simple-button": "SimpleElements/simple-button#^0.2.3"
+    "simple-button": "SimpleElements/simple-button#^0.2.3",
```

Comment:
I know I was the one who put this in here, but @seaneking we don't actually need `simple-button` at all do we?